### PR TITLE
DOC: update the pandas.Series.tail docstring

### PR DESCRIPTION
**Pull request by xrmx**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [X] PR title is "DOC: update the <your-function-or-method> docstring"
- [ ] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [X] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [X] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [X] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
################# Docstring (pandas.core.generic.NDFrame.tail) #################
################################################################################

Return the last n rows.

Parameters
----------
n : int, default 5
    Number of rows to select.

Returns
-------
type of caller
    The last `n` rows of the caller object.

See Also
--------
pandas.DataFrame.head : The first `n` rows of the caller object.

Examples
--------
>>> df = pd.DataFrame({'animal':['alligator', 'bee', 'falcon', 'lion',
...                    'monkey', 'parrot', 'shark', 'whale', 'zebra']})
>>> df
      animal
0  alligator
1        bee
2     falcon
3       lion
4     monkey
5     parrot
6      shark
7      whale
8      zebra

Viewing the last 5 lines

>>> df.tail()
   animal
4  monkey
5  parrot
6   shark
7   whale
8   zebra

Viewing the last n lines (three in this case)

>>> df.tail(3)
  animal
6  shark
7  whale
8  zebra

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	No extended summary found
```

Am not sure adding a summary would add much value.

### CODE REVIEW

File: pandas/core/indexes/base.py [hunk 1]
```diff
@@ -1156,7 +1156,7 @@ def to_frame(self, index=True):
         >>> idx = pd.Index(['Ant', 'Bear', 'Cow'], name='animal')
         >>> idx.to_frame()
             animal
-        animal       
+        animal
```
Comment (TomAugspurger) [inline pandas/core/indexes/base.py]: I thnk this was deliberate to pass the docstest. Can you revert this change?

**Comment by xrmx** [comment]
Updated PR, thanks!

**Comment by TomAugspurger** [comment]
Thanks!
